viewer/src/views/threejs/cad/CadNode.ts

```diff
@@ -176,33 +176,42 @@ export class CadNode extends THREE.Object3D {
   }
 
   private createLoadSectorsPipeline(): Subject<ThreeCameraConfig> {
+    const suspendLoading = this.loadingHints.suspendLoading;
+
     const loadSectorOperator = flatMap((s: WantedSector) => this._repository.loadSector(s));
     const consumeSectorOperator = flatMap((sector: ParsedSector) => this.rootSector.consumeSector(sector.id, sector));
+    function suspendIfLoadingToggled<T>(): OperatorFunction<T, T> {
+      return mergeMap<T, ObservableInput<T>>(x => (!suspendLoading ? of(x) : empty()));
+    }
 
     const pipeline = new Subject<ThreeCameraConfig>();
     pipeline
       .pipe(
         auditTime(100),
         fromThreeCameraConfig(),
+        suspendIfLoadingToggled(),
 
         // Determine all wanted sectors
         this._sectorCuller.determineSectors(),
 
         // Take sectors within budget
-        tap(c => console.log(`Want ${c.filter(x => x.levelOfDetail !== LevelOfDetail.Discarded).length} sectors`)),
+        // tap(c => console.log(`Want ${c.filter(x => x.levelOfDetail !== LevelOfDetail.Discarded).length} sectors`)),
```

Comment:
Ooops... I did not notice these getting pulled into master. These should probably be removed altogether (or at least hidden behind a debug flag).